src/main/java/org/opentripplanner/graph_builder/module/NearbyStopFinder.java

```diff
@@ -165,8 +183,24 @@ public NearbyStopFinder(Graph graph, double radiusMeters, boolean useStreets) {
                 if (targetVertex instanceof TransitStopVertex && state.isFinal()) {
                     stopsFound.add(StopAtDistance.stopAtDistanceForState(state, ((TransitStopVertex) targetVertex).getStop()));
                 }
+                if (OTPFeature.FlexRouting.isOn()
+                    && targetVertex instanceof StreetVertex
+                    && ((StreetVertex) targetVertex).flexStopLocations != null) {
+                   for (FlexStopLocation flexStopLocation : ((StreetVertex) targetVertex).flexStopLocations) {
```

Comment:
Add comment about "Simplification so that we only return one vertex from each stop location. Add all vertices tp the map, which is filtered below"